### DISCUSSION TITLE
Disable users from changing theme willy-nilly

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#20552] [Feature Requests]
### Summary

Initially the ability to change the Directus theme as an admin was a huge plus for us, but now for some reason it was felt that our users should also have that ability which is leading to them messing up the theme we spent hours on, and support tickets that just shouldn't be taking our time.

Please allow us to disable users from overwriting the theme.

![Screenshot 2023-11-28 at 08 23 01](https://github.com/directus/directus/assets/6641242/c6585163-baf9-4d28-8518-831ad1e6714a)

### Motivation

Working with css variables is not a common thing for daily business/content users to be doing. Not sure what the original motivation was for this, but it has zero value in all of our Directus projects

### Detailed Design

Simple remove the fields when feature is disabled.

### Must Have:
- Ability to remove feature from users

### Could Have:
- Ability to allow it per role

### Drawbacks

Not sure what the original motivation was.

### Alternatives

Hide the fields via injected css :(

### Adoption Strategy

None

**Comment by rijkvanzanten** [comment]
To me the improvement here is to _not_ require the theming stuff in the app required permissions, at which point you should be able to just remove it from the read access for the user 👍🏻 Lets track that as an improvement issue 🙂

**Comment by rijkvanzanten** [comment]
Ref https://github.com/directus/directus/issues/20558
